### ISSUE TITLE
6x6x6 step60 use FB prune table to avoid LR prune table solve "speed up IDA" call

### ISSUE THREAD

**Issue by dwalton76** [dwalton76/rubiks-cube-NxNxN-solver#13]
Today for 6x6x6 step60 we "solve" the self.lt_LR_solve_inner_x_centers_and_oblique_edges prune table to speed up the IDA search. When solving this prune table the IDA search is under 1s for cube `RBULFLLLDUFRLLDUFBFUUBUBFBFLLBUFDDFBRLRRDBRFUURUBURRBRBRUFBFLFDLRUBRUDFLLDLLLULBLDDBBLRRDDDLDLFDRUBUDDFRDBRRDDFUFDFLRRRULRUDFFRRFBLFDURBRRFUFLBFFBFRDBBBUFBDUDBLFLUBFLFRFDBDUBDBUULLDUBRDULFFRLUDBBDBLUDFLDBLDLRUUUUFRRR`
but if we don't solve the prune table first then IDA takes 3min 53s. I built a self.lt_FB_solve_inner_x_centers_and_oblique_edges.solve prune table but that only brought the time down to 3m 3s. 

So long story short, in order to produce a solution in a reasonable amount of time we need to solve that LR prune table.

**Comment by dwalton76** [comment]
won't fix
